cfme/tests/v2v/test_csv_import.py

```diff
@@ -269,17 +253,36 @@ def test_csv_archived_vm(appliance, infra_map, archived_vm):
     """Test csv with archived vm name
     Polarion:
         assignee: ytale
-        caseimportance: high
         caseposneg: positive
-        testtype: functional
         startsin: 5.10
         casecomponent: V2V
         customerscenario: true
         initialEstimate: 1/8h
-        subcomponent: RHV
-        upstream: yes
     """
     content = "Name\n{}".format(archived_vm)
     error_msg = "VM is inactive"
     assert import_and_check(appliance, infra_map, error_msg,
                             content=content, table_hover=True)
+
+
+@pytest.mark.uncollectif(lambda provider: provider.one_of(RHEVMProvider))
+def test_csv_security_group_flavor(appliance, infra_map, valid_vm, provider):
+    """Test csv with openstack security group and flavor
+    Polarion:
+        assignee: ytale
+        caseposneg: positive
+        startsin: 5.10
+        casecomponent: V2V
+        customerscenario: true
+        initialEstimate: 1/4h
+    """
+    security_group = provider.data.security_groups.admin[1]
+    flavor = provider.data.flavors[1]
+    content = "Name,Security Group,Flavor\n{valid_vm},{security_group},{flavor}".format(
+        valid_vm=valid_vm,
+        security_group=security_group,
+        flavor=flavor,
+    )
+    error_msg = {"security_group": security_group, "flavor": flavor}
```

Comment:
Question : Is this a positive test case ? on giving csv with security group and flavor do we get error message on UI ?

Reply:
Yes this is positive testcase and we are not checking error in UI. I used existing `error_msg` and `error_text` variables just to re-use `import_and_check` function. 

Basically we are checking whatever security group and flavor we provide via CSV import should be there on instance properties section of infra map.

Reply:
Yes this is positive testcase and we are not checking error in UI. I used existing `error_msg` and `error_text` variables just to re-use `import_and_check` function. 

Basically we are checking whatever security group and flavor we provide via CSV import (L281) should be there on instance properties section of infra map.